ui-tests/src/test/java/io/syndesis/qe/pages/integrations/ImportIntegrationPage.java

```diff
@@ -52,4 +66,89 @@ public void importIntegrationViaDragAndDrop(String integrationName) {
         getButton("Done").shouldBe(visible).click();
     }
 
+    public void testExportedFile(File file) {
+        try (ZipFile zipFile = new ZipFile(file)) {
+
+            Enumeration<? extends ZipEntry> entries = zipFile.entries();
+
+            while (entries.hasMoreElements()) {
+                ZipEntry zipEntry = entries.nextElement();
+                if (zipEntry.getName().equalsIgnoreCase("model.json")) {
+                    doTestExportedFile(zipFile, zipEntry);
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+            //TODO
+        }
+    }
+
+    private void doTestExportedFile(ZipFile file, ZipEntry entry) {
+        try (InputStream is = file.getInputStream(entry)) {
+            List<String> doc = new BufferedReader(new InputStreamReader(is,
+                    StandardCharsets.UTF_8)).lines().collect(Collectors.toList());
+
+            JSONObject jsonObj = new JSONObject(String.join("\n", doc));
+            JSONObject connections = jsonObj.getJSONObject("connections");
+
+            //I dont know the hash name of connection
+            Iterator<String> keys = connections.keys();
+            String currentConnectionType;
+            JSONObject currentConnectionJson;
+            String keyName;
+
+            while (keys.hasNext()) {
+                keyName = keys.next();
+                currentConnectionJson = connections.getJSONObject(keyName);
+                currentConnectionType = currentConnectionJson.getString("connectorId");
+
+                if (currentConnectionType.equalsIgnoreCase("twitter")) {
+                    testTwitterEncryption(currentConnectionJson);
+                } else if (currentConnectionType.equalsIgnoreCase("salesforce")) {
+                    testSalesforceEncryption(currentConnectionJson);
+                }
+                //TODO - create and export integration with all connections so we can test all of them here?
+                //TODO - is test for string ENC good enough? maybe test if it is not equal and get that specific secret from credentials.json
```

Comment:
You could try to compare with actual values from `AccountDirectory`. But ENC is fine for now imho.